docs/src/Faq.doc.js

```diff
@@ -210,6 +211,33 @@ yarn cypress run
   </Card>,
 );
 
+card(
+  <MainSection name="Icons and SVGs">
+    <MainSection.Subsection
+      description={`
+If you need a new icon for an experiment that is not listed on our [Icon](/Icon) documentation, use the \`dangerouslySetSvgPath\` prop on [Icon](/Icon), [IconButton](/IconButton), and [Pog](/Pog). However, \`dangerouslySetSvgPath\` only works with one SVG path. For icons with multiple paths and groups, use [Box](/Box) and \`dangerouslySetInlineStyle\` to pass the custom icon as \`backgroundImage\`.
+
+Once your experiment ships to 100%, ask your designer to follow the directions in the icon kit. Once the asset is ready, we can add the Icon to Gestalt.
```

Comment:
Can we link here to the Icon Kit?

Reply:
DONE https://www.figma.com/file/N60WnDx9j6Moz3Dt1rNsq9/Icon-Kit